com.unity.render-pipelines.high-definition/Documentation~/Refraction-in-HDRP.md

```diff
@@ -58,8 +58,7 @@ This means that, if screen space refraction does not return information for a pi
 
 The first tier of the refraction hierarchy is a screen space solution. To calculate screen space refraction, the algorithm traces a ray starting from the refractive object. It then refracts the ray according to the properties of the material. To compute the refracted ray, the algorithm assumes that the refractive object can be approximated as a simple shape ([Refraction Model](#RefractionModel)) .
 
-The refracted ray will be then intersected against the closest probe proxy volume to find the pixel in screen space that best approximates the result of the refracted ray. If there is no reflection probe proxy available, HDRP will fallback to a projection at infinite.
-
+The refracted ray intersects against the closest probe proxy volume to find the pixel in screen space that best approximates the result of the refracted ray. If there is no reflection probe proxy available, HDRP will use the renderer's bounding box to determine the proxy extents.
```

Comment:
Almos perfect, we just need to remove "will". 

"...HDRP uses the the renderer's bounding box to determine the proxy extents."